content/en/docs/setup/best-practices/certificates.md

```diff
@@ -95,6 +95,12 @@ Required certificates:
 | kube-apiserver-kubelet-client | kubernetes-ca             | system:masters | client           |                                                     |
 | front-proxy-client            | kubernetes-front-proxy-ca |                | client           |                                                     |
 
+{{< note >}}
+Instead of using the super-user group `system:masters` for `kube-apiserver-kubelet-client`
+a less privileged group can be used. kubeadm uses the `kubeadm:cluster-admins` group for
+that purpose.
+{{< /note >}}
```

Comment:
/lgtm

A question: Should we mention the behavior changed version v1.29 like the feature gate note?

Reply:
you mean the super-admin.conf feature in 1.29?
it was added without a feature gate.

Reply:
> you mean the super-admin.conf feature in 1.29?

I mean this note is since v1.29.

Reply:
well, this PR merges in the dev-1.29 branch and we don't have to do anything else, AFAIK.

the docs are versioned, so if users are on 1.28 they should be looking at 1.28 docs not at 1.29.

Reply:
i don't know what are the rules about these tags / notes